frontend/src/app/components/crm/leads/leads.component.html

```diff
@@ -61,6 +63,13 @@
 
         <mat-cell *matCellDef="let lead"> {{lead.columnData.PHONE_NUMBER}} </mat-cell>
       </ng-container>
+   <!-- Phone Number Column -->
```

Comment:
Change comment.

Reply:
just testing here. we don't need this column